src/entries/MyUserAccess.js

```diff
@@ -18,31 +19,29 @@ import '../App.scss';
 
 export const locale = 'en';
 
-const Routes = () => {
-  const history = useHistory();
+const MuaApp = () => {
+  const chrome = useChrome();
+  const navigate = useAppNavigate('/iam');
+  const userData = useUserData();
+
   useEffect(() => {
-    // redirect to MUA if url is "/settings"
-    if (window?.location?.pathname?.match(/\/(iam|settings)$/)) {
-      history.push('/my-user-access');
+    if (window?.location?.pathname?.match(/\/(iam)$/)) {
```

Comment:
We no longer need the window pathname reference we can use the location pathname as the history is shared with chrome.